DBB Cookbook/Business App Composition Recipes/BusinessApp_Dependencies.sql

```diff
@@ -0,0 +1,24 @@
+/*
+ * Business App Dependencies
+ * This is a simple query to quickly see what infrastructure impacts 1 or more Business App.
```

Comment:
Minor, should be `impacts one or more Business App`. Numerical values are typically only used for larger double digit values.